src/main/java/frc/robot/RobotContainer.java

```diff
@@ -28,21 +46,39 @@ public RobotContainer() {
     }
 
     private void configureDefaultCommands() {
-
+        swerve.setDefaultCommand(new DriveWithVision(swerve, vision, () -> target.getDegrees()));
     }
 
     private void configureButtonBindings() {
-
+        triggerLeft.whenPressed(() -> {
+            Robot.resetAngle();
+            swerve.resetThetaController();
+        });
+        trigger.whenPressed(() -> {
+            target = Robot.getAngle().minus(Rotation2d.fromDegrees(vision.getTargetYaw()));
+        });
     }
 
 
     /**
-    * Use this to pass the autonomous command to the main {@link Robot} class.
-    *
-    * @return the command to run in autonomous
-    */
+     * Use this to pass the autonomous command to the main {@link Robot} class.
+     *
+     * @return the command to run in autonomous
+     */
     public Command getAutonomousCommand() {
-      return null;
+        PathPlannerTrajectory path = PathPlanner.loadPath("Path", 3, 1.5, false);
+        Robot.resetAngle(path.getInitialState().holonomicRotation);
+        return new PPSwerveControllerCommand(
+                path,
+                swerve::getPose,
+                swerve.getKinematics(),
+                new PIDController(2, 0, 0),
+                new PIDController(2, 0, 0),
+                new ProfiledPIDController(2, 0, 0, new TrapezoidProfile.Constraints(3, 1.5)) {{
+                    enableContinuousInput(-Math.PI, Math.PI);
+                }},
```

Comment:
Just create a `ProfiledPIDController` object and call the method `enableContinuousInput`.
Also, add the PID coefficients to the Constants